src/Models/IoTProject.ts

```diff
@@ -698,31 +628,42 @@ export class IoTProject {
       throw error;
     }
 
-    const workspaceConfigFilePath = path.join(
-        this.projectRootPath,
-        `${path.basename(this.projectRootPath)}${
-            constants.workspaceConfigExtension}`);
+    const vscodeFolderPath = path.join(this.projectRootPath, FileNames.vscodeSettingsFolderName);
+    if (!fs.existsSync(vscodeFolderPath)) {
+      fs.mkdirSync(vscodeFolderPath);
+    }
+    const projectConfigFile = path.join(vscodeFolderPath, constants.projectConfigFileName);
+    if (!fs.existsSync(projectConfigFile)) {
+      fs.writeFileSync(projectConfigFile, JSON.stringify(projectConfig, null, 4));

```

Comment:
>4 [](start = 78, length = 1)

What's the magic number (4) here?

Reply:
Indentation space. Use const variable to better clarify the meaning.